rest/src/main/kotlin/rys/rest/exceptions/ChatNotFoundException.kt

```diff
@@ -0,0 +1,6 @@
+package rys.rest.exceptions
+
+class ChatNotFoundException : RuntimeException {
+    constructor(message: String?) : super(message)
+    constructor(message: String?, cause: Throwable?) : super(message, cause)
+}
```

Comment:
```suggestion
class ChatNotFoundException(message: String) : RuntimeException(message)
```

maybe this variant covers all your cases?